Compilation error: "no associated item named `Image` found"

Running `cargo run` on [this latest commit of my repository](https://github.com/Limeth/vulkanscratch/tree/db0bf3c9db8363193e3d6b120dd9422086545874) results in a compilation error originating from the `vulkano-shader-derive` macro.

```rust
$ cargo run
   Compiling vulkanscratch v0.1.0 (file:///home/limeth/workspace/rust/vulkanscratch)
error[E0599]: no associated item named `Image` found for type `vulkano::descriptor::descriptor::DescriptorDescTy` in the current scope
  --> src/main.rs:91:14
   |
91 |     #[derive(VulkanoShader)]
   |              ^^^^^^^^^^^^^

error: aborting due to previous error(s)

error: Could not compile `vulkanscratch`.

To learn more, run the command again with --verbose.
```

--- a/vulkano/src/descriptor/descriptor.rs
+++ b/vulkano/src/descriptor/descriptor.rs
@@ -105,7 +105,7 @@ impl DescriptorDesc {
 pub enum DescriptorDescTy {
     Sampler,                // TODO: the sampler has some restrictions as well
     CombinedImageSampler(DescriptorImageDesc),               // TODO: the sampler has some restrictions as well
-    ImageAccess(DescriptorImageDesc),
+    Image(DescriptorImageDesc),
     TexelBuffer {
         /// If `true`, this describes a storage texel buffer.
         storage: bool,
@@ -133,7 +133,7 @@ impl DescriptorDescTy {
         Some(match *self {
             DescriptorDescTy::Sampler => DescriptorType::Sampler,
             DescriptorDescTy::CombinedImageSampler(_) => DescriptorType::CombinedImageSampler,
-            DescriptorDescTy::ImageAccess(ref desc) => {
+            DescriptorDescTy::Image(ref desc) => {
                 if desc.sampled { DescriptorType::SampledImage }
                 else { DescriptorType::StorageImage }
             },
@@ -164,8 +164,8 @@ impl DescriptorDescTy {
             (&DescriptorDescTy::CombinedImageSampler(ref me),
              &DescriptorDescTy::CombinedImageSampler(ref other)) => me.is_superset_of(other),
 
-            (&DescriptorDescTy::ImageAccess(ref me),
-             &DescriptorDescTy::ImageAccess(ref other)) => me.is_superset_of(other),
+            (&DescriptorDescTy::Image(ref me),
+             &DescriptorDescTy::Image(ref other)) => me.is_superset_of(other),
 
             (&DescriptorDescTy::InputAttachment { multisampled: me_multisampled,
                                                   array_layers: me_array_layers },

